spec/install/typescript_spec.ts

```diff
@@ -1,5 +1,5 @@
-import {browser, by, By, element, $, $$, ExpectedConditions, protractor} from 'protractor/globals';
-import {ProtractorBrowser} from 'protractor';
+import {browser, by, By, element, $, $$, ExpectedConditions, protractor, ProtractorBrowser} from 'protractor';
```

Comment:
Reviewing this spec, it probably does not need `ProtractorBrowser`... lets remove it.